src/test/regress/expected/multi_extension_0.out

```diff
@@ -0,0 +1,702 @@
+--
```

Comment:
What do we need this file for? I think it's better to not have it and just tell devs to run `make install-all` from now on if they don't want `multi_extension` tests to fail.

Reply:
I did not want to disrupt the development of many developers. Is this really worth it?
I can easily drop the commit, just want to be sure.

Reply:
I think for stuff like HLL and tdigest it makes sense, since not everyone has those installed and it's some extra steps. But changing from `make install` to `make install-all` seems like a non-issue. Probably the test README and CONTRIBUTING should be updated though to show that you have to run this now.